### PR TITLE
Page breaks

### PR DESCRIPTION
**Pull request by brittany-johnson**
Link to spec docs: https://docs.google.com/spreadsheets/d/1vW0EHvn8fZgvTxZIm9xQXsqsnIDcolqwt5rWKtx4JEY/edit#gid=1804222758

more details: 
https://github.com/openstax/cnx-recipes/issues/861

### CODE REVIEW

File: styles/designs/common/pdf/_page-breaks.scss [hunk 1]
```diff
@@ -0,0 +1,78 @@
+//notes should be able to break but take the first element with them
```
Comment (helenemccarron) [inline styles/designs/common/pdf/_page-breaks.scss]: I really like the comment describing what the rules are, maybe we could have more of those so developers can refer to them later for ease and convenience?

File: styles/designs/common/pdf/_page-breaks.scss [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+nav#toc > ol > .os-toc-unit > ol > .os-toc-chapter > a > .os-number,
+nav#toc > ol > .os-toc-chapter > a > .os-number {
+  page-break-after: avoid;
+}
+
+div.preface {
+  page-break-before: right;
+  page-break-after: left;
+}
+
```
Comment (helenemccarron) [inline styles/designs/common/pdf/_page-breaks.scss]: I am not sure but I think we might want to add the rule to make sure the chapters like the index or appendices are starting on the left(?) side?

File: styles/designs/common/pdf/_page-breaks.scss [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  break-inside: avoid;
+}
+
+.learning-objectives {
+  page-break-inside: avoid;
+  page-break-before: avoid;
+}
+
+[data-type="note"], [data-type="example"] {
+  page-break-inside: auto;
```
Comment (helenemccarron) [inline styles/designs/common/pdf/_page-breaks.scss]: Is this necessary? This should be default, unless something else is setting it to a different value?

File: styles/designs/common/pdf/_page-breaks.scss (hunk 1 above)
Comment (brittany-johnson) [reply styles/designs/common/pdf/_page-breaks.scss]: I'm glad you pointed this out, this was actually a note for myself to implement this 😅

File: styles/designs/common/pdf/_page-breaks.scss (hunk 3 above)
Comment (brittany-johnson) [reply styles/designs/common/pdf/_page-breaks.scss]: Ahhh okay. This is one that I brought over from oer. It was probably in place to overwrite something else lol. I'll remove it!

File: styles/designs/common/pdf/_page-breaks.scss (hunk 1 above)
Comment (helenemccarron) [reply styles/designs/common/pdf/_page-breaks.scss]: Can you please open an issue for it, so we don't forget?

**Pull request merged**